cypress/models/loginPage.js

```diff
@@ -1,14 +1,16 @@
 import LoginPage from '../page-objects/login.page';
+import UniversalPages from '../page-objects/universal.pages';
```

Comment:
Mb better use CommonClass?

Reply:
I would like to make a universal class that can be used for all pages.